vmhost/hostCore/host.go

```diff
@@ -186,16 +207,16 @@ func NewVMHost(
 func createActivationMap(hostParameters *vmhost.VMHostParameters) map[uint32]struct{} {
 	activationMap := make(map[uint32]struct{})
 
-	activationMap[hostParameters.EnableEpochsHandler.CheckExecuteReadOnlyEnableEpoch()] = struct{}{}
-	activationMap[hostParameters.EnableEpochsHandler.DisableExecByCallerEnableEpoch()] = struct{}{}
-	activationMap[hostParameters.EnableEpochsHandler.RefactorContextEnableEpoch()] = struct{}{}
-	activationMap[hostParameters.EnableEpochsHandler.FixFailExecutionOnErrorEnableEpoch()] = struct{}{}
-	activationMap[hostParameters.EnableEpochsHandler.ManagedCryptoAPIEnableEpoch()] = struct{}{}
-	activationMap[hostParameters.EnableEpochsHandler.CreateNFTThroughExecByCallerEnableEpoch()] = struct{}{}
-	activationMap[hostParameters.EnableEpochsHandler.FixOOGReturnCodeEnableEpoch()] = struct{}{}
-	activationMap[hostParameters.EnableEpochsHandler.MultiESDTTransferAsyncCallBackEnableEpoch()] = struct{}{}
-	activationMap[hostParameters.EnableEpochsHandler.RemoveNonUpdatedStorageEnableEpoch()] = struct{}{}
-	activationMap[hostParameters.EnableEpochsHandler.StorageAPICostOptimizationEnableEpoch()] = struct{}{}
+	activationMap[hostParameters.EnableEpochsHandler.GetActivationEpoch(vmhost.CheckExecuteOnReadOnlyFlag)] = struct{}{}
+	activationMap[hostParameters.EnableEpochsHandler.GetActivationEpoch(vmhost.DisableExecByCallerFlag)] = struct{}{}
+	activationMap[hostParameters.EnableEpochsHandler.GetActivationEpoch(vmhost.RefactorContextFlag)] = struct{}{}
+	activationMap[hostParameters.EnableEpochsHandler.GetActivationEpoch(vmhost.FailExecutionOnEveryAPIErrorFlag)] = struct{}{}
```

Comment:
does `FixFailExecutionOnErrorEnableEpoch` equals with `FailExecutionOnEveryAPIErrorFlag` ?

Reply:
yes.. [proof of old method](https://github.com/multiversx/mx-chain-go/blob/rc/v1.6.0/common/enablers/enableEpochsHandler.go#L191)